test/e2e/scenarios/retina/retina_scenarios_test.go

```diff
@@ -0,0 +1,81 @@
+package retina
+
+import (
+	"os"
+	"os/user"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/microsoft/retina/test/e2e/framework/azure"
+	"github.com/microsoft/retina/test/e2e/framework/generic"
+	"github.com/microsoft/retina/test/e2e/framework/kubernetes"
+	"github.com/microsoft/retina/test/e2e/framework/types"
+	"github.com/microsoft/retina/test/e2e/scenarios/retina/drop"
+	tcp "github.com/microsoft/retina/test/e2e/scenarios/retina/tcp"
+)
+
+const (
+	// netObsRGtag is used to tag resources created by this test suite
+	netObsRGtag = "-e2e-netobs-"
+)
+
+// Test against AKS cluster with NPM enabled,
+// create a pod with a deny all network policy and validate
+// that the drop metrics are present in the prometheus endpoint
+func TestE2ERetinaMetrics(t *testing.T) {
+	job := types.NewJob("Validate that drop metrics are present in the prometheus endpoint")
+	runner := types.NewRunner(t, job)
+	defer runner.Run()
+
+	curuser, _ := user.Current()
```

Comment:
Any reason why we need this? Will this change often when running in similar agents?

Reply:
user will always be `runner` from gha, helps distinguish between runner rg's and user created rg's

name length is deterministic and in the current form shorter than ARM mandated 80 character limit